tasks/main.yml

```diff
@@ -10,12 +10,18 @@
     name: nginx-full
   when: ansible_pkg_mgr == "apt"
   
-- name: set nginx config
+- name: Set nginx config
   template:
     src: templates/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
   register: nginx_reload
 
+- name: Place template pam nginx file
+  template:
+    src: templates/pam_service_file.j2
+    dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```

Comment:
The var is not actually mandatory, as described in readme

Reply:
what are you talking about

Reply:
replace
```
dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```
by
```
dest: "/etc/pam.d/{{ nginx_pam_service_name | mandatory }}"
```

Reply:
done